aiidalab/app.py

```diff
@@ -0,0 +1,647 @@
+# -*- coding: utf-8 -*-
+"""Module to manage AiiDA lab apps."""
+
+import re
+import os
+from os import path
+from time import sleep
+from collections import OrderedDict
+
+import requests
+import ipywidgets as ipw
+from dulwich.repo import Repo
+
+
+class AppNotInstalledException(Exception):
+    pass
+
+
+class Version(ipw.VBox):
```

Comment:
```suggestion
class VersionSelector(ipw.VBox):
```
I think all widgets should have class names that clearly indicate that this is a widget.

Reply:
so should we maybe change it to `VersionSelectorWidget` instead?

Reply:
We had an offline discussion and agreed that the classes should be named according to the widely adopted camel case notation.

Reply:
done